### ISSUE TITLE
about second stage on nuscenes

### ISSUE THREAD

**Issue by qingzhouzhen** [tianweiy/CenterPoint#380]
I notice "Two stage refinement does not bring an improvement over the single-stage CenterPoint" in article, is there any specific experiment? for example they are exactly the same or low than one-stage?
@tianweiy

**Comment by tianweiy** [comment]
actually, we found that it somewhat works in our later paper. See table 3 of https://arxiv.org/pdf/2111.06881.pdf 

It is about 1 map improvement and not as significant as the improvement on Waymo probably due to the sparse point cloud in nuScenes

**Comment by qingzhouzhen** [comment]
> actually, we found that it somewhat works in our later paper. See table 3 of https://arxiv.org/pdf/2111.06881.pdf
> 
> It is about 1 map improvement and not as significant as the improvement on Waymo probably due to the sparse point cloud in nuScenes

Thanks for your reply, but MVP is multimodal and the pointcloud is not as sparse as original point cloud after adding virtual point  in nuScenes, So could you kindly share me CenterPoint's second stage performance in nuScenes ?

**Comment by tianweiy** [comment]
In that table, I have two stage for lidar-only.

**Comment by qingzhouzhen** [comment]
ok，thx



---Original---
From: "Tianwei ***@***.***&gt;
Date: Thu, Feb 16, 2023 21:37 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [tianweiy/CenterPoint] about second stage on nuscenes (Issue#380)




 
In that table, I have two stage for lidar-only.

**Comment by sankin97** [comment]
Have you been able to successfully replicate the two-stage performance? I’ve discovered that utilizing the ROI head for refinement results in a significant decrease in performance on the Nuscenes dataset.

**Comment by qingzhouzhen** [comment]
Yes I do，I get 1 percent increase confided as MVP with Voxelnet backbone on nuScenes.



---Original---
From: ***@***.***&gt;
Date: Sun, May 21, 2023 22:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [tianweiy/CenterPoint] about second stage on nuscenes (Issue#380)




 
Have you been able to successfully replicate the two-stage performance? I’ve discovered that utilizing the ROI head for refinement results in a significant decrease in performance on the Nuscenes dataset.

**Comment by sankin97** [comment]
How about the performance when not using MVP.
